src/create-thing.ts

```diff
@@ -0,0 +1,54 @@
+import { HomeAssistant, fireEvent } from 'custom-card-helpers';
```

Comment:
why not use the one in custom-card-helpers?

Reply:
I think I've changed something in the code to avoid having to use `custom` everywhere:
```js
  if (tag.startsWith('custom:'))
    tag = tag.substr('custom:'.length);
  else
    tag = `hui-${tag}-card`;
```

Reply:
Good call. We can add that to cch

Reply:
Just issued a PR. Not sure my dependencies didn't break anything though...

Reply:
published to npm; 1.2.7